test/CMakeLists.txt

```diff
@@ -1,14 +1,14 @@
-cmake_minimum_required(VERSION 3.8)
+cmake_minimum_required(VERSION 3.14...3.20)
 
-project(example_tests CXX)
+project(example_tests LANGUAGES CXX)
```

Comment:
Unnecessary change. `project`'s short form takes languages after the project name. Please read the documentation: https://cmake.org/cmake/help/latest/command/project.html